projects/ngx-prx-styleguide/src/lib/tz-datepicker/tz-datepicker.component.ts

```diff
@@ -70,9 +70,15 @@ export class TzDatepickerComponent implements OnInit {
   model: TzDate;
 
   _date: Date;
+  uninitialized = false;
   @Input()
   set date(value: Date) {
-    this._date = value;
+    if(value && value instanceof Date) {
+      this._date = value 
+    } else {
+      this._date = new Date()
```

Comment:
I _slightly_ worry about not being able to populate this field as blank/null?  It might be perfectly valid to have a blank in this field.  Won't setting it to `new Date()` have the impact of altering the model?

Reply:
Yeah, I can buy that this implementation is too tightly coupled to the way this component is currently used. It's currently unusable as an optional field. I'll take another crack at this (might be more appropriate to handle this in the consumer).

Reply:
I guess in my original comment, I was hoping this could be fixed in Publish alone.  By listening to the checkbox click/unclick events or something.